### ISSUE TITLE
RDS Role Configuration is supported in the docs but is rejected due to missing parameter on apply

### ISSUE THREAD

**Issue by Warfields** [hashicorp/terraform-provider-aws#15818]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version
Terraform 0.12.29
Provider is pinned at < 3.0.0

### Affected Resource(s)

* aws_rds_cluster

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp

resource "aws_rds_cluster" "amhub-db" {
  count = "${var.env_group == "anaprod" ? 1 : 0}"
  skip_final_snapshot                 = var.env_group == "anaprod"
  apply_immediately                   = var.env_group == "anaprod"
  engine                              = "aurora-postgresql"
  engine_version                      = "10.11"
  cluster_identifier                  = "amhub-db"
  database_name                       = var.rds_db_name
  master_username                     = var.rds_username
  master_password                     = module.vault_secret_delivery.secret_value
  backup_retention_period             = 7
  preferred_backup_window             = var.backup_window
  preferred_maintenance_window        = var.maintenance_window
  vpc_security_group_ids              = [aws_security_group.amhub-db-sg.id]
  db_subnet_group_name                = aws_db_subnet_group.amhub-db.name
  iam_database_authentication_enabled = true
  tags                                = module.amhub_tags.tags
}
```
### Expected Behavior

It should have attached the IAM Role
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster#iam_roles

### Actual Behavior
It creates a associated roles object that does not contain the feature-name parameter so AWS rejects the change

[Here's an example of what needs to be done](https://stackoverflow.com/questions/62798979/aurora-postgresql-permissions-to-access-s3)

Sample Output:
Error: InvalidParameterValue: The feature-name parameter must be provided with the current operation for the Aurora (PostgreSQL) engine.
        status code: 400, request id: 20748a3c-26a7-4ed8-af19-56a7ce620718

  on rds.tf line 85, in resource "aws_rds_cluster" "amhub-db-dev":
  85: resource "aws_rds_cluster" "amhub-db-dev" {

### Steps to Reproduce

1. [Use this parameter on a aws_rds_cluster](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster#iam_roles)
2. `terraform apply`

### References

* #0000

**Comment by nickmasster** [comment]
Probably a duplicate for #9552.

**Comment by breathingdust** [comment]
Hi @Warfields! 👋  Looks like this was resolved in #12370. Please comment to reopen if this is not the case! Thank you!
